scripts/config_comms_templates.py

```diff
@@ -0,0 +1,23 @@
+import requests
+from flask import json
+
+headers = {'Content-type': 'application/json'}
+
+with open('scripts/templates.json') as read_file:
+    contents = json.load(read_file)
+
+    region = "http://localhost:8182/classificationtypes/REGION"
+
+    legal_basis = "http://localhost:8182/classificationtypes/LEGAL_BASIS"
+
+    communication = "http://localhost:8182/classificationtypes/COMMUNICATION_TYPE"
+
+    requests.post(region, auth=('admin', 'secret'), headers=headers)
```

Comment:
I think we should be using  environment variables for this ?

Reply:
This script will be a one off so need to go in depth with it just to be ran once for this card